### DISCUSSION TITLE
exception: No module named 'sphinx_copybutton'

### DISCUSSION THREAD

**Discussion by fmrtl73** [executablebooks/meta#319] [Q&A]
**Describe the bug**

Could not import extension sphinx_copybutton (exception: No module named 'sphinx_copybutton')

**To Reproduce**
I followed the simple instructions to  `pip install sphinx-copybutton` and added the 'sphinx_copybutton' to my extensions: 
```
extensions = [
#    'sphinxmark',
    'sphinxcontrib.spelling',
    'sphinx_rtd_theme',
    'sphinx_copybutton',
]

When I run `tox -e docs` I get the following error:
```

Extension error:
Could not import extension sphinx_copybutton (exception: No module named 'sphinx_copybutton')
```
I also tried adding `sphinx_copybutton` to my requirements.txt but that did not help.

**Expected behavior**
It should just work as described in the docs.

**Environment**

- Python Version [e.g. 3.7.1]: 3.9
- Operating System: mac osx 10.15.7

**Comment by welcome** [comment]
Thanks for opening your first issue here! Engagement like this is essential for open source projects! :hugs:
<br>If you haven't done so already, check out [EBP's Code of Conduct](https://github.com/executablebooks/.github/blob/master/CODE_OF_CONDUCT.md). Also, please try to follow the issue template as it helps other community members to contribute more effectively.
<br>If your issue is a feature request, others may react to it, to raise its prominence (see [Feature Voting](https://executablebooks.org/en/latest/feature-vote.html)).
<br>Welcome to the EBP community! :tada:

**Comment by chrisjsewell** [comment]
Heya, what's in your tox.ini? This creates a separate virtual environment, and so if you haven't specified to install it in there, then it won't find it.
